data/benchmarking.ts

```diff
@@ -3,7 +3,7 @@
  * @difficulty beginner
  * @tags cli
  * @run deno bench <url>
- * @resource {https://deno.land/manual@v1.29.1/tools/benchmarker} Deno: Benchmarker tool
+ * @resource {https://deno.land/manual@v1.30.0/tools/benchmarker} Manual: Benchmarker tool
```

Comment:
How about removing this version number as well?

Reply:
We could do this, but the manual isn't stable yet. A few links actually had to be replaced with the closest thing I could find due to this. If that's okay with you, I'm game to remove the version number from the manual as well.